ldk/go/filesystemServer.go

```diff
@@ -181,3 +154,249 @@ func (f *FilesystemServer) FilesystemFileStream(req *proto.FilesystemFileStreamR
 	<-stream.Context().Done()
 	return nil
 }
+
+// FilesystemFileStream write data to a file
+func (f *FilesystemServer) FilesystemFileStream(stream proto.Filesystem_FilesystemFileStreamServer) error {
+	var file File
+	defer func() {
+		if file != nil {
+			file.Close()
+		}
+	}()
+
+	for {
+		select {
+		case <-stream.Context().Done():
+			return nil
+		default:
+			request, err := stream.Recv()
+			if err != nil {
+				return err
+			}
+
+			switch req := request.RequestOneOf.(type) {
+			case *proto.FilesystemFileStreamRequest_Open_:
```

Comment:
We're missing a branch for the Close request. Something like:

```suggestion+2
			case *proto.FilesystemFileStreamRequest_Close_:
			   return nil
```